### PR TITLE
[Update] add sort/limit/aggregate java example codes

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-get-data-from-single-table.md [hunk 1]
```diff
@@ -213,6 +250,38 @@ ORDER BY birth_year DESC
...
+            FROM authors
+            ORDER BY birth_year DESC
+            LIMIT ?;
+            """);
+        stmt.setInt(1, limit);
+        ResultSet rs = stmt.executeQuery();
+        while (rs.next()) {
+            Author author = new Author();
+            author.setId(rs.getLong("id"));
+            author.setName(rs.getString("name"));
```
Comment (Icemap) [inline develop/dev-guide-get-data-from-single-table.md:274]:
```suggestion
            author.setName(rs.getString("name"));
            author.setBirthYear(rs.getShort("birth_year"));
```

**Comment by Oreoxmt** [comment]
@Mini256 @septemberfd PTAL

**Review by Mini256 — APPROVED** [review]

File: develop/dev-guide-get-data-from-single-table.md [hunk 2]
```diff
@@ -170,16 +164,47 @@ public List<Author> getAuthorsByBirthYear(Short birthYear) throws SQLException {
 
 使用 `ORDER BY` 语句可以让查询结果按照期望的方式进行排序。
 
-例如，可以通过下面的 SQL 语句令 `authors` 表的数据根据 `birth_year` 列进行降序（`DESC`）排序，从而得到最年轻的作家列表。
+例如，可以通过下面的 SQL 语对 `authors` 表的数据按照 `birth_year` 列进行降序 (`DESC`) 排序，从而得到最年轻的作家列表。
```
Comment (Oreoxmt) [inline develop/dev-guide-get-data-from-single-table.md:167]:
```suggestion
例如，可以通过下面的 SQL 语句对 `authors` 表的数据按照 `birth_year` 列进行降序 (`DESC`) 排序，从而得到最年轻的作家列表。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**